doc/source/Linux.rst

```diff
@@ -0,0 +1,190 @@
+Linux Tutorial
+==============
+
+This guide gives you a brief introduction to how volatility3 works and some demonstration on suite of plugins available from
+
+Acquiring memory
+----------------
+
+Volatility does not provide the ability to acquire memory. In this tutorial we will see how we can use  `LiME <https://github.com/504ensicslabs/lime>`_ for this purpose. 
+It supports 32 and 64 bit captures from native Intel hardware as well as virtual machine guests. 
+It also supports capture from Android devices. See below for example commands building and running LiME:
+
+.. code-block:: shell-session
+
+    $ tar -xvzf lime-forensics-1.1-r14.tar.gz 
+    $ cd lime-forensics-1.1-r14/src
+    $ make
+    ....
+      CC [M]  /home/mhl/Downloads/src/tcp.o
+      CC [M]  /home/mhl/Downloads/src/disk.o
+    ....
+    $ sudo insmod lime-3.2.0-23-generic.ko "path=/home/mhl/ubuntu.lime format=lime"
+    $ ls -alh /home/mhl/ubuntu.lime 
+    -r--r--r-- 1 root root 2.0G Aug 17 19:37 /home/mhl/ubuntu.lime
+
+Procedure to create symbol tables for linux
+--------------------------------------------
+
+To create a symbol table please refer this :ref:`symbol-tables:Mac or Linux symbol tables`.
+We can also find some ISF files from `Linux ISF Server <https://isf-server.techanarchy.net/>`_ ,  which is built and maintained by `kevthehermit <https://twitter.com/kevthehermit>`_.
+After creating the file or downloading the file from the ISF server, please place the file under the directory ``volatility3/symbols/linux``. Make a directory linux under symbols.
+
+
+Listing plugins
+---------------
+
+Following are the list of linux plugins available for volatility3. More plugins will be available on future releases.
+For plugin requests, Please create an issue with description of the plugin.
+
+.. code-block:: shell-session
+
+    $ vol3 --help | grep -i linux. | head -n 5
```

Comment:
I think this would be good for unification.
```suggestion
    $ python3 vol.py --help | grep -i linux. | head -n 5
```


And commands over the pipeline are also very useful. However, I think it would be better to tell you the basic commands in the document as much as possible and omit them with the "..." mark. It is inefficient to show all results, but at least the user can see them through the command.

Reply:
Yes please!  This is the expected method of invocation, we can't guarantee the package unpacks as an executable.